lib/panel/index.ts

```diff
@@ -85,7 +86,12 @@ export default class Panel {
         }),
       )
 
-      await this.activate()
+      // Skip activating the panel if the setting "createPanelOnStart" has been unchecked
+      if (config.get('linter-ui-default.createPanelOnStart') as boolean) {
+        await this.activate()
+      }
+
+      this.initialized = true
```

Comment:
This means that the linter is initialized anyway.

Reply:
True, but not until after `this.activate()` has run. I found that the panel is activated twice really close together on startup because `this.activate()` calls `this.refresh()`, which then also calls `this.activate()`. The code added in `lib/panel/index.ts` on line 131 prevents `this.refresh()` from doing anything until `this.initialized` has been set, so the duplicate activation doesn't happen.